src/deburr.test.ts

```diff
@@ -0,0 +1,8 @@
+import assert from "node:assert";
+import test from "node:test";
+import {deburr} from "./deburr.js";
```

Comment:
importy sa snazim oddelovať to skupin node:*, node_module a local 

```suggestion
import assert from "node:assert";
import test from "node:test";

import { deburr } from "./deburr.js";
```

Reply:
[802e2db](https://github.com/xseman/bysquare/pull/32/commits/802e2db427abb1e1e5f3206b9104ed7c4d53ed70)

Poznamka: rozhodne som za poriadok v importoch. Bol by som vsak rad, keby to robil nejaky tool typu `prettier`. Verim, ze tak ostane viac priestoru pre iny typ feedbacku. Co si o tom myslis @xseman ?